hashes/src/hmac/mod.rs

```diff
@@ -15,15 +15,24 @@ use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use crate::{Hash, HashEngine};
 
 /// A hash computed from a RFC 2104 HMAC. Parameterized by the underlying hash function.
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Copy, Clone, PartialOrd, Ord, Hash)]
 #[repr(transparent)]
+#[allow(clippy::derived_hash_with_manual_eq)]
 pub struct Hmac<T: Hash>(T);
 
 impl<T: Hash + str::FromStr> str::FromStr for Hmac<T> {
     type Err = <T as str::FromStr>::Err;
     fn from_str(s: &str) -> Result<Self, Self::Err> { Ok(Hmac(str::FromStr::from_str(s)?)) }
 }
 
+impl<T: Hash> PartialEq for Hmac<T> {
+    fn eq(&self, other: &Self) -> bool {
+        crate::cmp::fixed_time_eq(self.as_ref(), other.as_ref())
+    }
```

Comment:
It'd be nice to have this documented since this may be a performance regression for some projects. They can use `.as_byte_array() == .as_byte_array()` of course, but there's currently no way for them to find out other than running benchmarks after update.